Source/WebCore/html/HTMLLinkElement.cpp

```diff
@@ -167,9 +166,13 @@ void HTMLLinkElement::setDisabledState(bool disabled)
 void HTMLLinkElement::parseAttribute(const QualifiedName& name, const AtomString& value)
 {
     if (name == relAttr) {
-        m_relAttribute = LinkRelAttribute(document(), value);
+        auto newRelAttribute = LinkRelAttribute(document(), value);
+        auto didMutateRel = newRelAttribute != m_relAttribute;
+        m_relAttribute = newRelAttribute;
         if (m_relList)
             m_relList->associatedAttributeValueChanged(value);
+        if (!didMutateRel)
```

Comment:
Not sure if early return is clearer here than just putting the call to `process()` inside an if statement. Something about the two return statements in a row looks a little inelegant.

Reply:
Yeah, I was thinking of the same so will change.